docker/testfiles/Dockerfile.ok

```diff
@@ -1 +1 @@
-FROM astronomerinc/ap-airflow:latest-onbuild
\ No newline at end of file
+FROM quay.io/astronomer/ap-airflow:latest-onbuild
```

Comment:
Is this auto-generated?  If not can you add EOF line?

Reply:
pre-commit would fix this automatically. Should we add a .pre-commit-config.yaml to the repo?

Reply:
yeah, maybe we can do that in a different PR.

Reply:
That makes sense. Also FYI I didn't change the existing line ending. It was missing before.

Reply:
I've got a PR open for pre-commit additions. https://github.com/astronomer/astro-cli/pull/380